src/components/list/RoomListItem.tsx

```diff
@@ -0,0 +1,24 @@
+import { RoomType } from '../../Interface';
+import { useNavigate } from 'react-router-dom';
+import { useRecoilValue } from 'recoil';
+import { userState } from '../../store/recoil';
+import { getUserInfoById } from '../../api';
+import { useCallback } from 'react';
+import ListItem from './ListItem';
+import React from 'react';
+
+const RoomListItem = ({ room }: { room: RoomType }) => {
+    const navigate = useNavigate();
+    const userData = useRecoilValue(userState);
+    const userId = room.user[1]['userId'];
```

Comment:
```suggestion
    const userId = room.user[1].userId;
```
가능하면 . selector를 사용하는것이 좋다고 알고있습니다.
(코딩 스타일 영역이라 단지 추천 입니다!)